gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/java/io/gravitee/rest/api/management/v2/rest/resource/api/analytics/ApiAnalyticsResource.java

```diff
@@ -91,4 +96,18 @@ public ApiAnalyticsAverageConnectionDurationResponse getAverageConnectionDuratio
             .map(ApiAnalyticsMapper.INSTANCE::map)
             .orElseThrow(() -> new NotFoundException("No connection duration found for api: " + apiId));
     }
+
+    @Path("/status-codes-by-entrypoint")
```

Comment:
```suggestion
    @Path("/response-statuses")
``` 
WDYT ? in accordance with my naming "refection"